.travis.yml

```diff
@@ -1,12 +1,22 @@
 sudo: false
-
 language: node_js
-
+before_install:
+- mkdir ./tests/configuration
+- ls ./tests
+- 'if [ "$TRAVIS_TRUSTED_INTERNAL_MERGE" = "true" ]; then openssl aes-256-cbc -K $encrypted_7e22a7c28f69_key -iv $encrypted_7e22a7c28f69_iv -in config.tar.enc -out ./tests/configuration/config.tar -d && tar xvf tests/configuration/config.tar; fi'
+- export GCLOUD_PROJECT=`cat tests/configuration/stub_project_id.txt`
+- export STUBBED_API_KEY=`cat tests/configuration/stub_api_key.txt`
+- export GOOGLE_APPLICATION_CREDENTIALS=$(pwd)/tests/configuration/stub_default_credentials.json
```

Comment:
@matthewloring as we discussed earlier today the only diag code being utilized here is just the bar minimum to get to the point of being able to make an authorized request and the authorization code itself is not directly tested by the error module.